toolkit/testconnection.php

```diff
@@ -5,6 +5,8 @@
 require_once(APPROOT.'core/utils.class.inc.php');
 require_once(APPROOT.'core/restclient.class.inc.php');
 
+Utils::$iConsoleLogLevel = Utils::ReadParameter('console_log_level', Utils::GetConfigurationValue('console_log_level', LOG_WARNING));
```

Comment:
I imagine that this is set because there are Utils::Log calls somewhere in the called methods stack ? 
I began to search but wasn't able to find :/

There's the same line added in dump_task.php
And this is already present in exec.php...
If we're duplicating code present elsewhere I would prefer to factorize it in a helper method ?

Reply:
Sure, fine for me. But is that to be done in this PR?

Reply:
As code duplication is done by this PR I would say, yes O:)
You could create a Utils method, maybe InitIConsoleLogLevel ?

Reply:
Okay, and that one would be called by `Utils::LoadConfig` or be placed instead of this current line we are now discussing?

Reply:
I was thinking of a simple helper to replace this single line. But we'll see with the rest of the dev team

Reply:
Okay, that sounds more logical indeed.

Reply:
Done.